configure.ac

```diff
@@ -38,6 +38,13 @@ AC_ARG_ENABLE(
 	[enable_distonly="no"]
 )
 
+AC_ARG_ENABLE(
+	[password-save],
+	[AS_HELP_STRING([--disable-password-save], [disable password save support @<:@default=yes@:>@])],
```

Comment:
I suppose you meant to write  "password-change", "password change support" etc.. and not "password save"